chapter_02_repository.asciidoc

```diff
@@ -325,7 +325,7 @@ def test_orderline_mapper_can_save_lines(session):
     fixtures for the purposes of this book, but the short explanation is that
     you can define common dependencies for your tests as "fixtures", and
     pytest will inject them to the tests that need them by looking at their
-    function arguments.  In this case, it's a SQLAlchemy database session.
+    function arguments.  In this case, it's an SQLAlchemy database session.
```

Comment:
haha, depends if you pronounce it "es-kew-ell-alchemy" or "sequel-alchemy", I guess!